source/Nevermore/Mapping/CustomTypeDefinition.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+
+namespace Nevermore.Mapping
+{
+    public abstract class CustomTypeDefinition : CustomTypeDefinitionBase
+    {
+        /// <summary>
+        /// Get the value to write to the database. Only override this in order to take over complete control of the
+        /// serialization for both columns and JSON.
+        /// </summary>
+        /// <param name="instance">The object instance being written</param>
+        /// <param name="isForJsonSerialization">True for serialization into the JSON column.</param>
+        /// <returns>The object value to write, or null to let the JsonSerializer handle the serialization.</returns>
+        public virtual object ToDbValue(object instance, bool isForJsonSerialization)
```

Comment:
I don't get this, why is it `ToDbValue` when it could be `isForJson`? 

I feel like I'd expect this type interface to have different methods for the different types of conversion.